src/integTest/groovy/net/wasdev/wlp/gradle/plugins/TestCompileJSP.groovy

```diff
@@ -18,19 +18,36 @@ public class TestCompileJSP extends AbstractIntegrationTest{
         createTestProject(buildDir, resourceDir, buildFilename)
     }
 
-    @AfterClass
-    public static void tearDown() throws Exception {
-        runTasks(buildDir, 'libertyStop')
+    @Test
+    public void check_for_jsp() {
+        try {
+            //runTasks(buildDir, 'compileJsp')
+        } catch (Exception e) {
+            throw new AssertionError ("Fail on task compileJsp. "+ e)
+        }
+        assert new File('build/testBuilds/test-compile-jsp/src/main/webapp/index.jsp').exists() : 'compileJsp Directory not found!'
     }
 
+
     @Test
     public void test_compile_jsp() {
         try {
-            runTasks(buildDir, 'libertyStart')
             runTasks(buildDir, 'compileJsp')
         } catch (Exception e) {
             throw new AssertionError ("Fail on task compileJsp. "+ e)
         }
+        assert(true)
         assert new File('build/testBuilds/test-compile-jsp/build/compileJsp').exists() : 'compileJsp Directory not found!'
     }
+
+
+    @Test
+    public void compile_jsp_class_file() {
+      try {
+          runTasks(buildDir, 'compileJsp')
+      } catch (Exception e) {
+          throw new AssertionError ("Fail on task compileJsp. "+ e)
+      }
+        assert new File('build/testBuilds/test-compile-jsp/build/classes/java/_index.class').exists() : '_index.class not found!'
+    }
```

Comment:
Is there a need to run compileJsp again or should this assert be moved to the previous test.  Is there a clean before the new compileJsp?  DId you figure out why your were timing out with a second compileJsp from the commandline yesterday?